controllers/disruption_controller.go

```diff
@@ -109,9 +109,15 @@ func (r *DisruptionReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 
 	// handle any chaos pods being deleted (either by the disruption deletion or by an external event)
 	if err := r.handleChaosPodsTermination(instance); err != nil {
-		r.log.Errorw("error handling chaos pods termination", "error", err)
+		if isModifiedError(err) {
+			r.log.Errorw("error handling chaos pods termination", "error", err)
```

Comment:
Do we want to log at the error level here? I thought we moved other logs to warn level but may be wrong?

Reply:
Oof, copy paste mistake

Reply:
Oof, more than just one copy paste mistake